server/Customer.java

```diff
@@ -22,30 +26,37 @@ public String accountsToString() {
 		return s;
 	}
 
-	public void addAccount(Account account) {
+	public void addAccount(newbank.server.Account account) {
 		accounts.add(account);
 	}
 
-	public ArrayList<Account> getAccounts() {
+	public ArrayList<newbank.server.Account> getAccounts() {
 		return accounts;
 	}
 
 	public boolean checkExistingAccount(String accountName) {
-		for (Account a : accounts) {
+		for (newbank.server.Account a : accounts) {
 			if (a.getAccountName().equals(accountName)) {
 				return true;
 			}
 		}
 		return false;
 	}
 
-	public Account getExistingAccount(String accountName) {
-		for (Account a : accounts) {
+	public newbank.server.Account getExistingAccount(String accountName) {
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it